### PR TITLE
Doxygen

### PR DESCRIPTION
**Pull request by awelch83**
I made changes to *.cpp files beginning with the letter 'A'. I updated comments, where necessary, to Doxygen standards. This is my first contribution, so I only wanted to touch a few files. If this is accepted, I'll know that what I did here was correct, and will do the same with other files.

### CODE REVIEW

**Comment by jonasschnelli** [comment]
@fanquake: can you have a look at this? IIRC, you are a doxygen guy?

**Comment by laanwj** [comment]
What is the result of this in the rendered doxygen?

**Comment by MarcoFalke** [comment]
Yes, please also include the diff in the *rendered* output of doxygen. Only changes to comments where you can influence the output meaningful are preferred.

**Comment by jamesob** [comment]
NACK. I doubt we want all these comments in the rendered doxygen output; most of these are useless if read anywhere but the source. Per [the developer notes](https://github.com/bitcoin/bitcoin/blob/master/doc/developer-notes.md#doxygen-comments), we only want to doxygen-document functions, methods, and fields.

File: src/addrman.cpp [hunk 1]
```diff
@@ -1,6 +1,6 @@
-// Copyright (c) 2012 Pieter Wuille
-// Distributed under the MIT software license, see the accompanying
-// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+/// Copyright (c) 2012 Pieter Wuille
+/// Distributed under the MIT software license, see the accompanying
+/// file COPYING or http://www.opensource.org/licenses/mit-license.php.
```
Comment (fanquake) [inline src/addrman.cpp]: We don't want any of these comments included in Doxygen, as they don't add value to the dev documentation.

File: src/addrman.cpp [hunk 2]
```diff
@@ -31,19 +31,24 @@ int CAddrInfo::GetBucketPosition(const uint256 &nKey, bool fNew, int nBucket) co
 
 bool CAddrInfo::IsTerrible(int64_t nNow) const
 {
-    if (nLastTry && nLastTry >= nNow - 60) // never remove things tried in the last minute
+    /// never remove things tried in the last minute
```
Comment (fanquake) [inline src/addrman.cpp]: As pointed out, we also don't want these comments included, as they make the most sense in the context of the code. I'll add some screenshots of what they look like in Doxygen to this thread.

**Comment by fanquake** [comment]
This is how the changed comments in Addrman appear.
doxygen

**Comment by laanwj** [comment]
Ok, closing this.
Blanketly making function-internal comments visible in doxygen can be confusing, and generates a way-too-large diff.

**Pull request closed without merging**